src/styles/product-card.scss

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  width: 200px;
```

Comment:
Create variables for all main values that can be changed/reused later (main sizes, colors, etc)